_posts/shane_wilson/2017-05-25-relay-migration.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title:  "Migrating to Relay Modern"
+breadcrumb: true
+author: shane_wilson
+date: 2017-05-25
+categories: shane_wilson
+tags:
+    - Relay
+    - React
+    - Javascript
+teaser:
+    info: A guide to migrating a legacy Relay code base to Relay Modern
+    image: shane_wilson/relay.png
+header:
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+This adds a bit more detail to the [Conversion Playbook](https://facebook.github.io/relay/docs/conversion-playbook.html){:target="_blank"}
```

Comment:
Could you add a short intro ? Idea is to give a bit more details about the context and give readers a general idea about what the blog post is about, even if they don't know the particular technologies involved.